content/docs/terminology.md

```diff
@@ -260,13 +260,13 @@ A filter is essentially an algebraic expression that must evaluate to`True` for
 Most commands accept filters.
 See [Filters](../filter/) for a more complete discussion.
 
-## gc
+## rebuilding
```

Comment:
Maybe an "a.k.a." is good here?

Reply:
If both terms "rc" and "rebuilding the working set' are used, then the docs should link them somewhere.